smartsim/launchable/baseJobGroup.py

```diff
@@ -8,14 +10,11 @@
 from .basejob import BaseJob
 
 
-class BaseJobGroup(Launchable, MutableSequence):
+class BaseJobGroup(Launchable, MutableSequence[BaseJob], ABC):
     """Highest level ABC of a group of jobs that can be
     launched
     """
 
-    def __init__(self) -> None:
```

Comment:
noted!

Reply:
This is 100% me forcing my style on the project, btw! These lines are perfectly valid, I just prefer to not redefine methods if all they are going to do is defer to the super implementation. I find it helps keep down the amount of clutter in a file.

If we decide that we actually prefer this style (I can't think of a good reason to prefer it off the top of my head, but that doesn't necessarily mean it doesn't exist) I could be swayed to revert this change!!